taichi/backends/metal/kernel_manager.cpp

```diff
@@ -681,7 +681,7 @@ class KernelManager::Impl {
     }
 
     if (pa->next >= shaders::kMetalPrintBufferSize) {
-      std::cout << "...(maximum print buffer reached)\n";
+      py_cout << "...(maximum print buffer reached)\n";
```

Comment:
Cool! FF2 merge this off-topic change if you did a careful test on Jupyter & IDLE.

Reply:
Ack, tested..